### DISCUSSION TITLE
v1.0.0-beta.3

### DISCUSSION THREAD

**Discussion by kamiya4047** [ExpTechTW/rts-map#8] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/ExpTechTW/rts-map/releases/tag/v1.0.0-beta.3'>v1.0.0-beta.3</a>.</em>

**Comment by kevin930321** [comment]
登入之後測站跟波形圖都是空的，請問要怎麼解決？![螢幕擷取畫面 2024-04-21 181520.png](https://github.com/ExpTechTW/rts-map/assets/78043580/d29dd6ef-9d2f-4444-818f-9c319ab6046e)

**Reply by shan33uwu** [reply]
同上
